core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/dataformats/DataFormatModel.java

```diff
@@ -72,7 +72,11 @@ public void setSupportedDataFormats(ArrayList<DataFormat> supportedDataFormats)
 	 * @return
 	 */
 	public DataFormat getDataFormatByName(String dataformat) {
-		return supportedDataFormats.stream().filter(df -> df.getName().equals(dataformat)).findFirst().orElse(null);
+		DataFormat foundDF = supportedDataFormats.stream().filter(df -> df.getName().equals(dataformat)).findFirst().orElse(null);
+		if (foundDF == null) {
+			foundDF = supportedDataFormats.stream().filter(df -> df.getModelName().equals(dataformat)).findFirst().orElse(null);
```

Comment:
I feel that it is not clean to take the first dataformat which is found but it seems that it not causing any issue.